std/hardcoded_strings/hardcoded_strings.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 ConsenSys Software Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by gnark DO NOT EDIT
+
+package hardcoded_strings
+
+import (
+	"fmt"
+	"github.com/consensys/gnark-crypto/ecc"
+	bls12_377 "github.com/consensys/gnark-crypto/ecc/bls12-377/fr"
+	bls12_381 "github.com/consensys/gnark-crypto/ecc/bls12-381/fr"
+	bls24_315 "github.com/consensys/gnark-crypto/ecc/bls24-315/fr"
+	bls24_317 "github.com/consensys/gnark-crypto/ecc/bls24-317/fr"
+	bn254 "github.com/consensys/gnark-crypto/ecc/bn254/fr"
+	bw6_633 "github.com/consensys/gnark-crypto/ecc/bw6-633/fr"
+	bw6_761 "github.com/consensys/gnark-crypto/ecc/bw6-761/fr"
+	"github.com/consensys/gnark/frontend"
+	"math/big"
+)
+
+func String(api frontend.API, str []byte) (frontend.Variable, error) {
```

Comment:
don't understand the package name and the function name -- nor while it is a fullblown package in the `std` ? a package in the `std`should read as "here is a circuit component that anyone can use"

Reply:
mmhh if it is something we want to do often, then it makes sense to expose it in its own package, but to me "hardcoded_strings.String" is not the right semantic. Maybe in a `bytes` package, so that it reads `bytes.ToVariable(api, b []byte)` ? Or, since it's applicable to constants only, a `constant` pacakge with `constant.FromString()` or `constant.FromByte()`. 

If we only use it at one place for now, I would make it package private there.

Reply:
I think `constant.FromString()` is the best option.